cypress/integration/advanced-functionalities/admin/system/role/Create.spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for creating a demo role', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for creating a demo role', () => {
+    it('should create a role', () => {
+      cy.visit(adminURL + '/')
+      // We wait 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('.nav-sidebar').contains('Roles').click()
+      cy.get('[data-test-id="button-new-role"]').click()
+      cy.get('[data-test-id="card-role-info"]').within(() => {
+        cy.get('[data-test-id="input-name"]').type('Demo')
+        cy.get('[data-test-id="input-handle"]').type('demo')
+        cy.get('[data-test-id="textarea-description"]').type('This is a role created for demonstration.')
```

Comment:
to demonstrate what? Not sure if this is the right description, name and handle for the role